deploy.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import shlex
+import subprocess
+import sys
+
+
+def main(*argv):
+    parser = argparse.ArgumentParser(
+        description="Prep a release of the `conda-forge`-based installer."
+    )
+    parser.add_argument(
+        "-f", "--force", action="store_const",
+        default="", const="--force",
+        help="Forcibly tag and push (if `--push` given)."
```

Comment:
Provide the option to forcibly overwrite the tag and forcibly push. Should ideally not be used. However if things get really messed up, it is a nice option to have.